src/Widgets/GeneralSection.vala

```diff
@@ -54,7 +54,27 @@ public class MouseTouchpad.Widgets.GeneralSection : Gtk.Grid {
         attach (primary_button_switcher, 1, 1, 2, 1);
         attach (new SettingLabel (_("Reveal pointer:")), 0, 2, 1, 1);
         attach (reveal_pointer_switch, 1, 2, 1, 1);
-        attach (help_icon, 2, 2, 1, 1);
+        attach (locate_pointer_help, 2, 2, 1, 1);
+
+        var interface_settings_schema = SettingsSchemaSource.get_default ().lookup ("org.gnome.desktop.interface", false);
+
```

Comment:
You need to test whether "null" has been returned before using interface_settings_schema.

I got a crash running this branch on Loki (may or may not be related).